docs/src/model_setup/output_writers.md

```diff
@@ -178,7 +177,7 @@ c_avg = Field(Average(model.tracers.c, dims=(1, 2)))
 
 # Note that model.velocities is NamedTuple
 simulation.output_writers[:velocities] = JLD2OutputWriter(model, model.velocities,
-                                                          prefix = "some_more_data",
+                                                          filename = = "some_more_data.jld2",
```

Comment:
```suggestion
                                                          filename = "some_more_data.jld2",
```

Reply:
Similarly going down in this file.

Reply:
Oops. Thanks!

Reply:
Actually I'd fixed that already. I think you made a comment on an old version...